### PR TITLE
Jv change plop warn to debug

### PR DESCRIPTION
**Pull request by JoukoVirtanen**
## Checklist
- [x] Investigated and inspected CI test results

## Testing Performed

CI is sufficient

### CODE REVIEW

File: central/processlisteningonport/datastore/datastore_impl.go [hunk 1]
```diff
@@ -129,7 +129,7 @@ func (ds *datastoreImpl) AddProcessListeningOnPort(
 		// and will not be stored.
 		if _, indicatorExists := indicatorsMap[indicatorID]; !indicatorExists {
 			countMetrics.IncrementOrphanedPLOPCounter(val.GetClusterId())
-			log.Warnf("Found no matching indicators for %+v", val)
+			log.Debugf("Found no matching indicators for %+v", val)
```
Comment (dashrews78) [inline central/processlisteningonport/datastore/datastore_impl.go:132]: Even in debug I don't think we can log the `process_args` as those could contain secrets.  We should find the subset of information that tells us something without anything sensitive.

Comment (JoukoVirtanen) [reply central/processlisteningonport/datastore/datastore_impl.go]: Done. `process_args` are no longer being logged here.

Comment (JoukoVirtanen) [reply central/processlisteningonport/datastore/datastore_impl.go]: I removed logging of `process_args` from everywhere.

**Review by dashrews78 — APPROVED** [review]
LGTM

**Pull request merged**